### ISSUE TITLE
Item interactions not working properly/missing

### ISSUE THREAD

**Issue by cringoleg** [MarkusBordihn/BOs-Easy-Mob-Farm#43]
**Describe the bug**
Interactions with glass bottle slot in farm blocks are somewhat broken:
- You cannot insert multiple glass bottles with manual & fast insert (shift + LMB).
- After inserting a single glass bottle and waiting a bit, you can notice that amount of `Bottle o' Enchanting` exceeds amount of glass bottles you put there previously (one). This leads to getting an unreasonable amount of XP for basically free - one glass bottle :)

**To Reproduce**
Try to insert multiple amount of glass bottles into farm.

**Expected behavior**
1. A reasonable implementation would likely include the multi-item interactions i.e. `shift` + `LMB` from player's inventory into farm (with stacked glass bottles).

**Some ideas on related functionality**
1. Hoppers would be an awesome solution to automating inserting/replacing glass bottles to `Bottle o' Enchanting`  slot.
1.1 Like in `hopper -> furnace` interaction, from the **top side** of farm block insert swords, from **the horizontal** sides insert glass bottles or vice-versa.
2. This one is more of a *nice touch* - if the inserted sword has the `mending` enchantment, remove the possible XP drop to bottle and repair the sword for the respective amount, while still counting `tool damage` to the sword.

**Additional context**
Minecraft version: `1.19.4`
Mod version: `6.7.0`

Mod List:
```
backpacked-forge-1.19.4-2.1.14.jar
balm-forge-1.19.4-6.0.2.jar
cgm-forge-1.19.4-1.3.7.jar
chunkloaders-1.2.8-forge-mc1.19.3.jar
collective-1.19.4-6.66.jar
corpse-1.19.4-1.0.2.jar
easy_mob_farm_1.19.4-6.6.2.jar
enchantmenttransfer-0.0.6-1.19.4.jar
everythingcopper-1.19.4-0.2.2.0.jar
framework-forge-1.19.4-0.6.16.jar
improvedmobs-1.19.4-1.11.5-forge.jar
inventorysorter-22.0.0.jar
jei-1.19.4-forge-13.1.0.16.jar
lootr-1.19.4-0.6.27.65.jar
merenc-forge-1.20.x-v3.2.jar
NaturesCompass-1.19.4-1.11.0-forge.jar
netherportalfix-forge-1.19.4-12.0.1.jar
Stabx_Modern_Guns_v6.3_1.19.4_Final.jar
supermartijn642configlib-1.1.8-forge-mc1.19.jar
supermartijn642corelib-1.1.15-forge-mc1.19.4.jar
tenshilib-1.19.4-1.7.0-forge.jar
treeharvester-1.19.4-8.2.jar
YungsApi-1.19.4-Forge-3.10.1.jar
YungsBetterDungeons-1.19.4-Forge-3.4.0.jar
YungsBetterMineshafts-1.19.4-Forge-3.4.0.jar
YungsBetterOceanMonuments-1.19.4-Forge-2.3.0.jar
YungsBetterStrongholds-1.19.4-Forge-3.4.0.jar
YungsBridges-1.19.4-Forge-3.3.0.jar
YungsExtras-1.19.4-Forge-3.3.0.jar
```

**Comment by MarkusBordihn** [comment]
Thanks for your feedback. It's by design that you can only insert 1 bottle at a time, to avoid the need to have an additional slot like bottle input slot and bottle output slot.

Furthermore technically you are getting a lot of more XP from a furnace farm or a real mob farm than from this farm.
So I would not see this as a concern, because there are other way to get much more XP in a faster time without any mods.
Anyways I'm happy to implement a config for the XP drops so that you have more control about this like:

- 1 glass bottle only 1 bottle of Bottle o' Enchanting
- Number of XP per XP drop
- Possibility of XP per mob drop

Let me know if this sounds good or if you have any other ideas for an easy to use xp drop system.

The other things are already implemented, but not in an overpowered way.
Mending weapon get's repaired but it will not be enough to repair your weapon for free.
Using mending and unbreaking will eventually work, but there is no guarantee that you weapon will stay forever.
If the weapon is fully repaired you could also get out XP bottles.

Automation is already implemented, you can insert empty bottles, swords and mob catcher over an hopper and extract the mob loot and Bottle o' Enchanting over another hopper. I will add a wiki page for this to explain this more in detail.

**Comment by cringoleg** [comment]
Thank you for explanation Markus!
For me the XP question was just about the bottles. If I can automate bottle I/O operations then everything is good. I have tried automations but they didn't work for some reason... Will be waiting for those!
Nevertheless I would like to try out some XP configs if you have them in your sprint/plan.
I and my friends appreciate your work!

Sorry about the mending - should've tried it first :D

**Comment by MarkusBordihn** [comment]
Here is an simple example with the automation:
https://github.com/MarkusBordihn/BOs-Easy-Mob-Farm/wiki/Automation

Let me know if there is anything which is not working correctly.

I have no XP config changes on my plan but it's always a good thing to allow users to customize settings as much as possible instead of using some hard-coded magic. ;)

**Comment by cringoleg** [comment]
That's really good! Thank you.
I will test it tomorrow, if anything comes up - I will report new issues and mention them in this thread.

**Comment by MarkusBordihn** [comment]
This should be fixed with 6.9.0 which should be live in the next hours.
